File: Week 1/Problem Set 1/1. mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // List of variables
    int n;
    int x;
    int y;
    int z;

    x = 0;
    y = 0;

    // Declaration of n, limited beetwen 1 and 8
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // Initialization of z to be modified in the loop
    z = n;

    // Mario loop !
    for (int i = 0; i < n; i++)
    {
        x = y;

        for (int j = z - 2; j >= 0; j--)
        {
            printf(" ");
        }
        for (x = y; x >= 0; x--)
        {
            printf("#");
        }
        printf("  ");
        for (x = y; x >= 0; x--)
        {
            printf("#");
        }
        printf("\n");
        z--;
        y++;
    }
}

File: Week 6/Problem Set 6/1. sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    # List of variables
    n = 0
    y = 0
    z = 0

    # Declaration of n, limited beetwen 1 and 8
    while True:
        n = get_int("Height: ")
        if n >= 1 and n <= 8:
            break

    # Initialization of z to be modified in the loop
    z = n

    # Mario loop !
    for i in range(n):
        x = y
        for j in range(z - 2, -1, -1):
            print(" ", end="")
        for x in range(y, -1, -1):
            print("#", end="")
        print("  ", end="")
        for x in range(y, -1, -1):
            print("#", end="")
        print()
        z -= 1
        y += 1


main()
